packages/data-point/lib/core/core.js

```diff
@@ -93,6 +89,11 @@ function create (spec) {
   manager.addEntityTypes(options.entityTypes, true)
   manager.addEntities(options.entities)
 
+  // supports currying
+  manager.resolve = Transform.resolve(manager)
+  // does not support currying
+  manager.transform = _.partial(Transform.transform, manager)
```

Comment:
I debated between adding curry to transform, I think we should leave it as is, no need to add more breaking changes, hope u guys agree